src/docs/product/alerts-notifications/alerts/index.mdx

```diff
@@ -0,0 +1,115 @@
+
+When an alert is created or changes status, the actions associated with the trigger are executed. The available actions are:
+
+- Send an email (to a member or team). If sent to a member, the member's personal project alert opt-out settings are overridden.
+- Send a [Slack](/product/integrations/slack/) notification.
+- Send a [PagerDuty](/product/integrations/pagerduty/) incident.
+- Send a request to a webhook (via [internal integrations](/product/integrations/integration-platform/#internal-integrations)).
+
+### Alert Stream
+
+The Alert Stream displays your metric alerts along with visual graphs, associated projects, and the last time the alert was triggered. Access alerts via the **Alerts** sidebar item.
+
+![Alert stream illustrates alert duration, trends, severity, time duration and percentage of users affected.](alert-stream2.png)
+
+### Alert Details
+
+The Alert Details page provides specifics on a single instance of alert from its creation to its resolution, or the current time if the alert is still active. The "subscribe" button subscribes you to workflow notifications for the alert, including status changes and comments.
+
+![Alert details illustrates a graph showing errors.](alert_details.png)
+
+## Issue Alerts
+
+Issue alerts apply only to event errors, and fire whenever any issue in the project matches the specified criteria, such as a resolved issue re-appearing or an issue affecting many users.
+
+### Conditions
+
+Conditions are evaluated for an issue alert **each time** the issue receives a new event, subject to [rate limits](/product/alerts-notifications/alerts/#rate-limit). Each condition is **evaluated independently** of other conditions. For example, the following alert will never trigger:
+
+![The alert condition in this example shows the conditions are conflicting with each other.](alert-conditions2.png)
+
+In the example, the alert will not fire because an event **cannot satisfy both** of these conditions simultaneously: Sentry detects/creates a new issue, _and_ the issue has happened ten times.
+
+For a complete list of conditions, see [Alert Conditions](/product/alerts-notifications/alerts/alert-conditions).
+
+### Actions
+
+Actions are the configurations that determine what will happen when an alert is triggered. The following actions are available:
+
+- Send an email to either [Issue Owners](/product/error-monitoring/issue-owners/), Team, or [Member](/accounts/membership/#member)
+- Send a notification to an [integration](/product/integrations/), which includes [PagerDuty](/product/integrations/integrations/pagerduty/), [Slack](/product/integrations/integrations/slack/), and [Microsoft Teams](/product/integrations/integrations/msteams/). If no legacy integrations or integrations built using the integration platform are enabled, this option is hidden.
```

Comment:
this list might be useful for line 24, perhaps

Reply:
@adhiraj do these conditions also apply to Metric Alerts?